views/containers.html

```diff
@@ -1,7 +1,7 @@
 <div ng-if="!containers.length"><em>none</em></div>
 <dl class="dl-horizontal" ng-repeat="container in containers">
 <dt>Name</dt>
-<dd>{{container.name}}</dd>
```

Comment:
this extension won't be available in this project... there are several things (extension points, utility filters, etc), that we need to think through how we manage